kuma/wiki/jinja2/wiki/includes/buttons.html

```diff
@@ -37,7 +37,9 @@
         </div>
       </li>{% endif %}
 
+      {% if (not document.is_template) or (document.is_template and user.is_authenticated() and user.has_perm('wiki.change_template_document')) %}
```

Comment:
The test ``user.is_authenticated()`` is unnecessary. The [AnonymousUser](https://docs.djangoproject.com/en/1.10/ref/contrib/auth/#django.contrib.auth.models.AnonymousUser) will return ``False`` for ``user.has_perm('wiki.change_template_document')``. So, a less verbose, equivalent condition would be:

```
     {% if (not document.is_template) or user.has_perm('wiki.change_template_document') %}
```